### PR TITLE
Add Pluto TV prefix

### PR DESCRIPTION
**Pull request by freearhey**
Adds the prefix `Pluto TV` to the name of channels broadcast from the domain `pluto.tv`.

### CODE REVIEW

**Comment by themak1985** [comment]
I think this approach is fine, but should it be "Channel Name (Pluto TV)"? Your approach is nice, because you can see all the Pluto Channels in one place when sorting alphabetically, but some pluto hosted channels are regular channels, like Cheddar.

As for pluto tv being international, not really. Yes they are not geo-restricted, but that does not make them INT.
If you use the pluto tv app or website, change your vpn to UK, Italy, Mexico, etc. You will see all the channels change. They have different channels for different markets.

File: channels/unsorted.m3u [hunk 1]
```diff
@@ -551,15 +551,15 @@ https://stitcheraws.unreel.me/wse-node02.powr.com/live/5c95396f932c837b49397360/
 http://bazarhd.online:80/307210113151/307210113151/80392
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",BEIN SPORTS 2 UHD
 http://bazarhd.online:80/307210113151/307210113151/80391
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",beIN SPORTS XTRA
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV beIN SPORTS XTRA
```
Comment (themak1985) [inline channels/unsorted.m3u:554]: Pluto Stream of Regular Channel

File: channels/unsorted.m3u [hunk 2]
```diff
@@ -605,25 +605,25 @@ https://bibeltv02.iptv-playoutcenter.de/bibeltv02/bibeltv02.stream_1/playlist.m3
 http://bibeltv03.iptv-playoutcenter.de/bibeltv03/bibeltv03.stream_1/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Big Life TV
 https://biglife.sinclair.wurl.com/manifest/3000.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Big Sky Conference
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV Big Sky Conference
```
Comment (themak1985) [inline channels/unsorted.m3u:608]: Pluto Stream of Regular Channel (I think?)

File: channels/unsorted.m3u [hunk 3]
```diff
@@ -641,7 +641,7 @@ https://tv.ensonhaber.com/tv/tr/bloomberght/index.m3u8
 https://d1g4mbj10o50ni.cloudfront.net/qt/Channel-QT-TX-AWS-virginia-2/Source-QT-10M-2-sn9jy9-BP-07-02-oNnPi5xc6sq_live.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Bloomberg Quicktake
 https://d1g4mbj10o50ni.cloudfront.net/qt/Channel-QT-TX-AWS-virginia-2/Source-QT-10M-2-sn9jy9-BP-07-03-qPp9HnlfNtK_live.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Bloomberg TV
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV Bloomberg TV
```
Comment (themak1985) [inline channels/unsorted.m3u:644]: Pluto Stream of Regular Channel

File: channels/unsorted.m3u [hunk 4]
```diff
@@ -919,25 +919,25 @@ http://cdn2.envivo.win/HLS/?canal=zonafilmstv&c=ZwNjYwRkBF4kBQLhZmt=&p=ZGH4AGD1Z
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV CBSN 2
 https://service-stitcher.clusters.pluto.tv/stitch/hls/channel/5a6b92f6e22a617379789618/master.m3u8?terminate=false&deviceType=web&deviceMake=web&deviceModel=web&sid=204&deviceId=5a6b92f6e22a617379789618&deviceVersion=DNT&appVersion=DNT&deviceDNT=0&userId=&advertisingId=&deviceLat=&deviceLon=&app_name=&appName=web&buildVersion=&appStoreUrl=&architecture=&includeExtendedEvents=false&marketingRegion=US&serverSideAds=false
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",CBSN Bay Area
 https://dai.google.com/linear/hls/event/VE8b4n-YStusEGv5Z2NmsQ/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",CBSN INTERNACIONALES
 https://cbsnhls-i.akamaihd.net/hls/live/264710/cbsn_hlsprod_2/master_360.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",CBSN Los Angeles
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV CBSN Los Angeles
```
Comment (themak1985) [inline channels/unsorted.m3u:936]: Pluto Stream of Regular Channel

File: channels/unsorted.m3u [hunk 5]
```diff
@@ -919,25 +919,25 @@ http://cdn2.envivo.win/HLS/?canal=zonafilmstv&c=ZwNjYwRkBF4kBQLhZmt=&p=ZGH4AGD1Z
...
 https://service-stitcher.clusters.pluto.tv/stitch/hls/channel/5dc481cda1d430000948a1b4/master.m3u8?terminate=false&deviceType=web&deviceMake=web&deviceModel=web&sid=207&deviceId=5dc481cda1d430000948a1b4&deviceVersion=DNT&appVersion=DNT&deviceDNT=0&userId=&advertisingId=&deviceLat=&deviceLon=&app_name=&appName=web&buildVersion=&appStoreUrl=&architecture=&includeExtendedEvents=false&marketingRegion=US&serverSideAds=false
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",CBSN Minnesota
 https://dai.google.com/linear/hls/event/zcWPVCfURNSPxeidcckQLA/master.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",CBSN New York
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV CBSN New York
```
Comment (themak1985) [inline channels/unsorted.m3u:940]: Pluto Stream of Regular Channel

File: channels/unsorted.m3u [hunk 6]
```diff
@@ -993,19 +993,19 @@ https://stream-us-east-1.getpublica.com/playlist.m3u8?network_id=68&live=1&app_b
...
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/hJxFLvF.png" group-title="",Channel S (US Pacific)
 https://a.jsrdn.com/broadcast/22679_24MrQma9TX/-0800/c.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Chassy
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV Chassy
 https://service-stitcher.clusters.pluto.tv/stitch/hls/channel/5b6b285823ecb93393cbf766/master.m3u8?terminate=false&deviceType=web&deviceMake=web&deviceModel=web&sid=687&deviceId=5b6b285823ecb93393cbf766&deviceVersion=DNT&appVersion=DNT&deviceDNT=0&userId=&advertisingId=&deviceLat=&deviceLon=&app_name=&appName=web&buildVersion=&appStoreUrl=&architecture=&includeExtendedEvents=false&marketingRegion=US&serverSideAds=false
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Cheddar
 https://live.chdrstatic.com/cheddar/primary/5.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Cheddar
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV Cheddar
```
Comment (themak1985) [inline channels/unsorted.m3u:1000]: Pluto Stream of Regular Channel

File: channels/unsorted.m3u [hunk 7]
```diff
@@ -1079,49 +1079,49 @@ http://cmctv.ios.internapcdn.net/cmctv_vitalstream_com/live_1/CMCU-92/chunklist.
...
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",CN 3
 https://wowza.telpin.com.ar:1935/canal3/canal3.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",CNBC Indonesia
 https://live.cnbcindonesia.com/livecnbc/smil:cnbctv.smil/chunklist_w576315509_b384000_sleng.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",CNET
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV CNET
 https://service-stitcher.clusters.pluto.tv/stitch/hls/channel/56283c8769ba54637dea0464/master.m3u8?terminate=false&deviceType=web&deviceMake=web&deviceModel=web&sid=228&deviceId=56283c8769ba54637dea0464&deviceVersion=DNT&appVersion=DNT&deviceDNT=0&userId=&advertisingId=&deviceLat=&deviceLon=&app_name=&appName=web&buildVersion=&appStoreUrl=&architecture=&includeExtendedEvents=false&marketingRegion=US&serverSideAds=false
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",CNN
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV CNN
```
Comment (themak1985) [inline channels/unsorted.m3u:1090]: Pluto Stream of Regular Channel

File: channels/unsorted.m3u [hunk 8]
```diff
@@ -1497,7 +1497,7 @@ https://zypelive-lh.akamaihd.net/i/default_1@710948/index_3800_av-p.m3u8?F38b1dF
 https://srvx1.selftv.video/espansione/live/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS-ikVAyjIFBMuH1_tknzbL5zxfU9kQw1YfPw&usqp=CAU" group-title="",ESTACION TV HD
 http://unlimited1-us.dps.live/estaciontv/estaciontv.smil/estaciontv/livestream2/chunks.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",ET Live
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV ET Live
```
Comment (themak1985) [inline channels/unsorted.m3u:1500]: Pluto Stream of Regular Channel

File: channels/unsorted.m3u [hunk 9]
```diff
@@ -1672,7 +1672,7 @@ http://htv-drm-live-cdn.fptplay.net/CDN-FPT02/FOXMOVIES-HD-1080p/playlist.m3u8
 https://livecdn.fptplay.net/foxlive/foxmovieshd_hls.smil/chunklist_b2500000.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="http://static.epg.best/us/FoxNews.us.png" group-title="News",Fox News(美国)
 http://1111296894.rsc.cdn77.org/ls-54548-2/tracks-v1a1/mono.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",FOX Sports
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV FOX Sports
```
Comment (themak1985) [inline channels/unsorted.m3u:1675]: Pluto Stream of Regular Channel

File: channels/unsorted.m3u [hunk 10]
```diff
@@ -2454,7 +2454,7 @@ http://5tv.lincoln.ne.gov/live/WIFI-2096k-1080p/WIFI-2096k-1080p.m3u8
 http://80tv.lincoln.ne.gov/live/WIFI-2096k-1080p/WIFI-2096k-1080p.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",LNKTV Health
 http://10tv.lincoln.ne.gov/live/WIFI-2096k-1080p/WIFI-2096k-1080p.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Logo Pluto TV
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV Logo Pluto TV
```
Comment (themak1985) [inline channels/unsorted.m3u:2457]: Double name?

File: channels/unsorted.m3u [hunk 11]
```diff
@@ -2628,7 +2628,7 @@ http://hlslive-akc-med1.media.mlb.com/ls01/mlbnetwork/NETWORK_LINEAR_1/MLBN_FAUX
 http://hlslive-akc-med1.media.mlb.com/ls01/mlbnetwork/NETWORK_LINEAR_2/master_wired.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcSZXP7luKnB2atWBF9dbb-r4ceCj2sOxFQ0y4NqpZ2jwLOcuWq9&usqp=CAU" group-title="",MLB TV
 http://hlslive-akc-med1.media.mlb.com/ls01/mlbnetwork/NETWORK_LINEAR_2/MLBN_FAUX_LINEAR_STREAM_2_MLBN_Linear_Stream_Friday_20200612_1591985931119/1800K/1800_slide.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",MLS
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV MLS
```
Comment (themak1985) [inline channels/unsorted.m3u:2631]: Pluto Stream of Regular Channel?

File: channels/unsorted.m3u [hunk 12]
```diff
@@ -2652,49 +2652,49 @@ https://a.jsrdn.com/broadcast/0c9a09c94c/+0000/low/c.m3u8
...
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",MTV pluto.tv
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV MTV pluto.tv
 https://service-stitcher.clusters.pluto.tv/stitch/hls/channel/5caf325764025859afdd6c4d/master.m3u8?terminate=false&deviceType=web&deviceMake=web&deviceModel=web&sid=1b711cc8-0587-11eb-9df2-0242ac110002&deviceId=5caf325764025859afdd6c4d&deviceVersion=DNT&appVersion=DNT&deviceDNT=1&userId=&advertisingId=&deviceLat=&deviceLon=&app_name=&appName=web&buildVersion=&appStoreUrl=&architecture=&includeExtendedEvents=false&marketingRegion=DE&serverSideAds=false
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",MTV pluto.tv+
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV MTV pluto.tv+
```
Comment (themak1985) [inline channels/unsorted.m3u:2689]: Double name?

File: channels/unsorted.m3u [hunk 13]
```diff
@@ -2652,49 +2652,49 @@ https://a.jsrdn.com/broadcast/0c9a09c94c/+0000/low/c.m3u8
...
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",MTV Latino
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV MTV Latino
 https://service-stitcher.clusters.pluto.tv/stitch/hls/channel/5cf96d351652631e36d4331f/master.m3u8?terminate=false&deviceType=web&deviceMake=web&deviceModel=web&sid=965&deviceId=5cf96d351652631e36d4331f&deviceVersion=DNT&appVersion=DNT&deviceDNT=0&userId=&advertisingId=&deviceLat=&deviceLon=&app_name=&appName=web&buildVersion=&appStoreUrl=&architecture=&includeExtendedEvents=false&marketingRegion=US&serverSideAds=false
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",MTV pluto.tv
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Pluto TV MTV pluto.tv
```
Comment (themak1985) [inline channels/unsorted.m3u:2687]: Double Name

File: channels/us.m3u [hunk 14]
```diff
@@ -531,21 +531,21 @@ https://dai.google.com/linear/hls/event/26FJK7wRSo6RhPsK70XS_w/master.m3u8
...
 #EXTINF:-1 tvg-id="CBSNDallas.us" tvg-name="CBSN Dallas" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5eceb0d4065c240007688ec6/colorLogoPNG.png" group-title="",CBSN Dallas
 http://dai.google.com/linear/hls/event/o5J3g4U9T16CvYnS7Qd86Q/master.m3u8
-#EXTINF:-1 tvg-id="CBSNDenver.us" tvg-name="CBSN Denver" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5eb1b12146cba40007aa7e5d/colorLogoPNG.png" group-title="",CBSN Denver
+#EXTINF:-1 tvg-id="CBSNDenver.us" tvg-name="CBSN Denver" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5eb1b12146cba40007aa7e5d/colorLogoPNG.png" group-title="",Pluto TV CBSN Denver
```
Comment (themak1985) [inline channels/us.m3u:542]: Pluto Stream of Regular Channel

File: channels/us.m3u [hunk 15]
```diff
@@ -531,21 +531,21 @@ https://dai.google.com/linear/hls/event/26FJK7wRSo6RhPsK70XS_w/master.m3u8
...
+#EXTINF:-1 tvg-id="CBSNBoston.us" tvg-name="CBSN Boston" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5eb1af2ad345340008fccd1e/colorLogoPNG.png" group-title="",Pluto TV CBSN Boston
 https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/5eb1af2ad345340008fccd1e/master.m3u8?deviceId=channel&deviceModel=web&deviceVersion=1.0&appVersion=1.0&deviceType=rokuChannel&deviceMake=rokuChannel&deviceDNT=1&advertisingId=channel&embedPartner=rokuChannel&appName=rokuchannel&is_lat=1&bmodel=bm1&content=channel&platform=web&tags=ROKU_CONTENT_TAGS&coppa=false&content_type=livefeed&rdid=channel&genre=ROKU_ADS_CONTENT_GENRE&content_rating=ROKU_ADS_CONTENT_RATING&studio_id=viacom&channel_id=channel
-#EXTINF:-1 tvg-id="CBSNChicago.us" tvg-name="CBSN Chicago" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5eb1aeb2fd4b8a00076c2047/colorLogoPNG.png" group-title="",CBSN Chicago
+#EXTINF:-1 tvg-id="CBSNChicago.us" tvg-name="CBSN Chicago" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5eb1aeb2fd4b8a00076c2047/colorLogoPNG.png" group-title="",Pluto TV CBSN Chicago
```
Comment (themak1985) [inline channels/us.m3u:538]: Pluto Stream of Regular Channel

File: channels/us.m3u [hunk 16]
```diff
@@ -531,21 +531,21 @@ https://dai.google.com/linear/hls/event/26FJK7wRSo6RhPsK70XS_w/master.m3u8
...
+#EXTINF:-1 tvg-id="CBSNBayArea.us" tvg-name="CBSN Bay Area" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5eb1afb21486df0007abc57c/colorLogoPNG.png" group-title="",Pluto TV CBSN Bay Area
 https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/5eb1afb21486df0007abc57c/master.m3u8?deviceId=channel&deviceModel=web&deviceVersion=1.0&appVersion=1.0&deviceType=rokuChannel&deviceMake=rokuChannel&deviceDNT=1&advertisingId=channel&embedPartner=rokuChannel&appName=rokuchannel&is_lat=1&bmodel=bm1&content=channel&platform=web&tags=ROKU_CONTENT_TAGS&coppa=false&content_type=livefeed&rdid=channel&genre=ROKU_ADS_CONTENT_GENRE&content_rating=ROKU_ADS_CONTENT_RATING&studio_id=viacom&channel_id=channel
-#EXTINF:-1 tvg-id="CBSNBoston.us" tvg-name="CBSN Boston" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5eb1af2ad345340008fccd1e/colorLogoPNG.png" group-title="",CBSN Boston
+#EXTINF:-1 tvg-id="CBSNBoston.us" tvg-name="CBSN Boston" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5eb1af2ad345340008fccd1e/colorLogoPNG.png" group-title="",Pluto TV CBSN Boston
```
Comment (themak1985) [inline channels/us.m3u:536]: Pluto Stream of Regular Channel

File: channels/us.m3u [hunk 17]
```diff
@@ -531,21 +531,21 @@ https://dai.google.com/linear/hls/event/26FJK7wRSo6RhPsK70XS_w/master.m3u8
 https://dai.google.com/linear/hls/event/Sid4xiTQTkCT1SLu6rjUSQ/master.m3u8
 #EXTINF:-1 tvg-id="CBSNBayArea.us" tvg-name="CBSN Bay Area" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/PNrkhuf.jpg" group-title="Local",CBSN Bay Area (720p)
 https://dai.google.com/linear/hls/event/VE8b4n-YStusEGv5Z2NmsQ/master.m3u8?iu=/4128/CBS.SF.OTT
-#EXTINF:-1 tvg-id="CBSNBayArea.us" tvg-name="CBSN Bay Area" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5eb1afb21486df0007abc57c/colorLogoPNG.png" group-title="",CBSN Bay Area
+#EXTINF:-1 tvg-id="CBSNBayArea.us" tvg-name="CBSN Bay Area" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5eb1afb21486df0007abc57c/colorLogoPNG.png" group-title="",Pluto TV CBSN Bay Area
```
Comment (themak1985) [inline channels/us.m3u:534]: Pluto Stream of Regular Channel

File: channels/us.m3u [hunk 18]
```diff
@@ -585,7 +585,7 @@ https://live.chdrstatic.com/cheddar/primary/1.m3u8
 https://live.chdrstatic.com/cheddar/primary/index.m3u8
 #EXTINF:-1 tvg-id="Cheddar.us" tvg-name="Cheddar" tvg-country="US" tvg-language="English" tvg-logo="http://miro.medium.com/max/2656/1*DhgWWj7DTZ-kD2zWhBBwAg.png" group-title="Business",Cheddar
 https://livestream.chdrstatic.com/b93e5b0d43ea306310a379971e384964acbe4990ce193c0bd50078275a9a657d/cheddar-42620/cheddarweblive/cheddar/primary/4.m3u8
-#EXTINF:-1 tvg-id="Cheddar.us" tvg-name="Cheddar" tvg-country="US" tvg-language="English" tvg-logo="https://images.pluto.tv/channels/5ca1e02c08ee5378be82db47/colorLogoPNG.png" group-title="",Cheddar
+#EXTINF:-1 tvg-id="Cheddar.us" tvg-name="Cheddar" tvg-country="US" tvg-language="English" tvg-logo="https://images.pluto.tv/channels/5ca1e02c08ee5378be82db47/colorLogoPNG.png" group-title="",Pluto TV Cheddar
```
Comment (themak1985) [inline channels/us.m3u:588]: Pluto Stream of Regular Channel

File: channels/us.m3u [hunk 19]
```diff
@@ -597,35 +597,35 @@ https://live.chdrstatic.com/cbn/primary/index.m3u8
...
 https://service-stitcher.clusters.pluto.tv/stitch/hls/channel/5dcb62e63d4d8f0009f36881/master.m3u8?advertisingId=&appName=web&appVersion=5.14.0-0f5ca04c21649b8c8aad4e56266a23b96d73b83a&app_name=web&clientDeviceType=0&clientID=6fbead95-26b1-415d-998f-1bdef62d10be&clientModelNumber=na&deviceDNT=false&deviceId=6fbead95-26b1-415d-998f-1bdef62d10be&deviceLat=19.4358&deviceLon=-99.1441&deviceMake=Chrome&deviceModel=web&deviceType=web&deviceVersion=88.0.4324.150&marketingRegion=VE&serverSideAds=false&sessionID=b8e5a857-714a-11eb-b532-0242ac110002&sid=b8e5a857-714a-11eb-b532-0242ac110002&userId=
-#EXTINF:-1 tvg-id="CINEACCIONPLUTOTV.us" tvg-name="CINE ACCION PLUTO.TV" tvg-country="US" tvg-language="Spanish" tvg-logo="https://images.pluto.tv/channels/5dcb62e63d4d8f0009f36881/colorLogoPNG.png?w=117" group-title="",CINE ACCION PLUTO.TV
+#EXTINF:-1 tvg-id="CINEACCIONPLUTOTV.us" tvg-name="CINE ACCION PLUTO.TV" tvg-country="US" tvg-language="Spanish" tvg-logo="https://images.pluto.tv/channels/5dcb62e63d4d8f0009f36881/colorLogoPNG.png?w=117" group-title="",Pluto TV CINE ACCION PLUTO.TV
```
Comment (themak1985) [inline channels/us.m3u:610]: Double name

File: channels/us.m3u [hunk 20]
```diff
@@ -597,35 +597,35 @@ https://live.chdrstatic.com/cbn/primary/index.m3u8
...
 https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/5d8d164d92e97a5e107638d2/master.m3u8?deviceId=channel&deviceModel=web&deviceVersion=1.0&appVersion=1.0&deviceType=rokuChannel&deviceMake=rokuChannel&deviceDNT=1&advertisingId=channel&embedPartner=rokuChannel&appName=rokuchannel&is_lat=1&bmodel=bm1&content=channel&platform=web&tags=ROKU_CONTENT_TAGS&coppa=false&content_type=livefeed&rdid=channel&genre=ROKU_ADS_CONTENT_GENRE&content_rating=ROKU_ADS_CONTENT_RATING&studio_id=viacom&channel_id=channel
-#EXTINF:-1 tvg-id="CINEDRAMAPLUTOTV.us" tvg-name="CINE DRAMA PLUTO.TV" tvg-country="US" tvg-language="Spanish" tvg-logo="https://images.pluto.tv/channels/5b4e92e4694c027be6ecece1/featuredImage.jpg?fill=blur&fit=fill&fm=jpg&h=900&q=75&w=1600" group-title="",CINE DRAMA PLUTO.TV
+#EXTINF:-1 tvg-id="CINEDRAMAPLUTOTV.us" tvg-name="CINE DRAMA PLUTO.TV" tvg-country="US" tvg-language="Spanish" tvg-logo="https://images.pluto.tv/channels/5b4e92e4694c027be6ecece1/featuredImage.jpg?fill=blur&fit=fill&fm=jpg&h=900&q=75&w=1600" group-title="",Pluto TV CINE DRAMA PLUTO.TV
```
Comment (themak1985) [inline channels/us.m3u:612]: Double Name

File: channels/us.m3u [hunk 21]
```diff
@@ -924,21 +924,21 @@ https://gma2.blab.email/espnews.m3u8
...
 #EXTINF:-1 tvg-id="ETLive.us" tvg-name="ET Live" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/vTlQAIX.jpg" group-title="Entertainment",ET Live (1080p)
 https://etlive-mediapackage-fastly.cbsaavideo.com/dvr/manifest.m3u8?iu=/8264/vaw-can/desktop/cbslocal
-#EXTINF:-1 tvg-id="ETLive.us" tvg-name="ET Live" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5dc0c78281eddb0009a02d5e/colorLogoPNG.png" group-title="",ET Live
+#EXTINF:-1 tvg-id="ETLive.us" tvg-name="ET Live" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5dc0c78281eddb0009a02d5e/colorLogoPNG.png" group-title="",Pluto TV ET Live
```
Comment (themak1985) [inline channels/us.m3u:935]: Pluto Stream of Regular Channel

**Comment by freearhey** [comment]
@themak1985 Thanks for the feedback. I didn't know that. Do you know if there is some simple way to determine if a channel is original or only by comparing feeds?

**Comment by themak1985** [comment]
I took a look, I can't find a clear differentiator.

**Comment by freearhey** [comment]
Got it.

**Pull request merged**